.github/workflows/_ci.yaml

```diff
@@ -476,15 +476,78 @@ jobs:
   #     ARTIFACTS: |
   #       test-equinox.log
   #   secrets: inherit
+  test-te-h100:
+    needs: build-jax
+    if: inputs.ARCHITECTURE == 'amd64'
+    uses: ./.github/workflows/_transformer_engine_eks.yaml
+    with:
+      JAX_IMAGE: ${{ needs.build-jax.outputs.DOCKER_TAG_FINAL }}
+      JOB_NAME: transformerengine-${{ github.run_id }}
+      S3_BUCKET: jax-toolbox-eks-output
+      CI_NAME: transformer-engine
+    secrets: inherit
 
-  # test-te-multigpu:
-  #   needs: build-upstream-pax
-  #   if: inputs.ARCHITECTURE == 'amd64' # arm64 runners n/a
-  #   uses: ./.github/workflows/_test_te.yaml
-  #   with:
-  #     TE_IMAGE: ${{ needs.build-upstream-pax.outputs.DOCKER_TAG_FINAL }}
-  #   secrets: inherit
+  test-te-multigpu-a100:
```

Comment:
I didn't spot any coverage that this is adding in addition to test-te-unit-a100?